.github/workflows/linux.yml

```diff
@@ -0,0 +1,44 @@
+name: linux
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: (${{ matrix.target }}, ${{ matrix.channel }}, ${{ matrix.cfg-release-channel }})
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [
+          x86_64-unknown-linux-gnu,
+        ]
+        channel: [ nightly ]
+        cfg-release-channel: [
+          beta,
+          nightly,
+        ]
+
+    env:
+      CFG_RELEASE_CHANNEL: ${{ matrix.cfg-release-channel }}
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+      # Run build
+    - name: setup
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ matrix.channel }}-${{ matrix.target }}
+        target: ${{ matrix.target }}
+        override: true
+        profile: minimal
+        default: true
+    - name: build
+      run: |
+        rustc -Vv
+        cargo -V
+        cargo build
+    - name: test
+      run: cargo test
+    - name: 'test ignored'
```

Comment:
I'll update this (and the others) to account for the recent change splitting into multiple crates